### PR TITLE
Pull Request template

### LINKED ISSUE #64: FEAT [0015]
# INDEX
1 [REQUIREMENT](#requirement)

1.1 [USER STORIES](#user-stories)

1.2 [WORKFLOW](#workflow)

1.3 [REQUIREMENTS (DETAILS)](#requirement-details)

1.4 [WIREFRAMES](#wireframes)

2 [RESOURCES](#resources)

2.1 [TOOLS](#tools)

2.2 [SUGGESTED IMPLEMENTATION](#suggested-implementation)

2.3 [REFERENCES](#references)

3 [GLOSSARY](#glossary)

# REQUIREMENT

## USER STORIES
This states all of the scenarios of the users involved.

- [x] #65 
- [x] #66 

## WORKFLOW
This should include a picture of the screens involved. 
Error states and view changes based on role should be documented. 
This picture is worth a thousand words, as the details of the flow through the feature can be quite complex, and this is hard to spell out the details in the next section. We have been using Gliffy, a confluence plugin, and it rocks. It is a flash app that allows us to create and edit the workflows with eases.

## REQUIREMENT DETAILS
These are the details of the feature. Document all screen(s) and every field, label, validation, message, and action. This is essentially the functional specification of the details of the screen(s) involved. Because it is in the context of the wireframe (next section), it is more concise. You can simply reference the field name, rather than verbosely state everything about the field. You can keep the details to field length, required, etc.

## WIREFRAMES
A picture is required for each screen involved. Wireframes can be simple drawings on a whiteboard that are photographed, or a set of boxes created in Visio or OmniGraffle (or anything really). Some are photoshop-ed, and others are HTML. We have found that the best approach is HTML if that’s your end result. Recreating HTML that looks the exact same as a picture can be quite painful. Usually, a visual designer is involved in creating the mock ups, and can easily do this work in HTML. The developers can therefore take the exact HTML that is requested, and fill in their code as needed.

*******

# RESOURCES

## SUGGESTED IMPLEMENTATION
Some code or pseudo-code
```

```
## SUGGESTED TOOLS
PLUGINS, LIBRARIES, FRAMEWORKS, ETC
1. 
2.
3.

## REFERENCES

- [Some external reference](https://github.com)

*******

# GLOSSARY

### LINKED ISSUE #65: As a [SOME ROLE] I want to [DO SOMETHING] So that I [CAN GET SOME BENEFIT]
Feature Specification: #64 

### ACCEPTANCE CRITERIA

GIVEN that condition 1 and condition 2….
WHEN I do step 1, and step 2…
THEN, desired result 1, desired result 2….

### LINKED ISSUE #66: As a [SOME ROLE] I want to [DO SOMETHING] So that I [CAN GET SOME BENEFIT]
Feature Specification #64 

### ACCEPTANCE CRITERIA

GIVEN that condition 1 and condition 2….
WHEN I do step 1, and step 2…
THEN, desired result 1, desired result 2….

### PR DESCRIPTION
**Pull request by polusasciuro**
## Types of changes
What types of changes does your code introduce?

 

 

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
- [ ] Documentation Update (if none of the other choices apply)

## What is the current behavior? 

- closes #64 
- closes #66 
- clsoes #65 

## Test plan
Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by ecureuill — COMMENTED** [review]
Some doubt???

**Comment by polusasciuro** [comment]
> Some doubt???

bla bla bla

**Review by ecureuill — CHANGES_REQUESTED** [review]
> / job01
> 
> .github#L1
> project [PULL REQUEST CYCLE] is not found.

File: .github/ISSUE_TEMPLATE/BUG_NEXT_TEMPLATE.MD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-name: Bug Next report
+name: 07 - Bug report in release
```
Comment (ecureuill) [inline .github/ISSUE_TEMPLATE/BUG_NEXT_TEMPLATE.MD:2]:
```suggestion
name: 07 - Bug report in release
```

File: .github/ISSUE_TEMPLATE/BUG_NEXT_TEMPLATE.MD [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 about: Use this template to track bugs in next release
-labels: bug on next release,0 - Backlog
+labels: bug on next release
```
Comment (ecureuill) [inline .github/ISSUE_TEMPLATE/BUG_NEXT_TEMPLATE.MD:4]:
```suggestion
labels: bug on next release
```

File: .github/ISSUE_TEMPLATE/DESIGN_NOTE_TEMPLATE.MD [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+name: 05 - Design Note
+about: Use this template for create a DESIGN NOTE for a feature.
+labels: Feature Design Notes, 1 - Planning
```
Comment (ecureuill) [inline .github/ISSUE_TEMPLATE/DESIGN_NOTE_TEMPLATE.MD:4]:
```suggestion
labels: Feature Design Notes, 1 - Planning
```

**Review by ecureuill — CHANGES_REQUESTED** [review]
not working

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,167 @@
+name: main
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  issues:
+    types: [edited, labeled, unlabeled]
+  pull_request:
+    types: [labeled, unlabeled]
```
Comment (ecureuill) [inline .github/workflows/main.yml]:
```suggestion
    types: [opened]
```

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+
+#     needs: [triage, assign_one_project]
+      - name: commenter by label
+        uses: peaceiris/actions-label-commenter@v1.3.2 
+#      #Label triggered GitHub Actions for posting a template message, automatically close or reopen issues or pull requests
+        if: github.event.action == 'labeled' || github.event.action == 'unlabeled' 
+        with:
+           github_token: ${{ secrets.GITHUB_TOKEN }}
+           config_file: .github/label-commenter-config01.yml
```
Comment (ecureuill) [inline .github/workflows/main.yml]:
```suggestion
           config_file: .github/label-commenter-config.yml
```

**Review by ecureuill — CHANGES_REQUESTED** [review]

**Review by ecureuill — APPROVED** [review]

**Pull request merged**